### ISSUE TITLE
Problem 4 (counterFactory) should we have example returns after like the previous problem?

### ISSUE THREAD

**Issue by mikkelrd** [DevMountain/JS-Closures#11]
@JRobber These instructions refer to the "module pattern" but don't really explain what that means. Furthermore, that is a vague term. A student also pointed out that in programming you usually know what you want the outcome to be and you use your tools to get there, so he would have appreciated example return values on this one just like the previous one, like so:
```
var counter = counterFactory(10);
counter.inc(); // 11
counter.dec(); // 10
counter.dec(); // 9
```

**Comment by TomPridham** [comment]
I also had a lot of people be confused by that. It seems like the lectures only touch on it for a bit, if I remember correctly. A link to a good, succinct description of the module pattern would prolly be good. http://stackoverflow.com/questions/17776940/javascript-module-pattern-with-example?answertab=votes#tab-top ?

**Comment by asBrettisay** [comment]
I second @TomPridham, we should have a good answer to what the module pattern is, that's something that students should probably understand before they finish.

**Comment by asBrettisay** [comment]
Addressed in #16 

I added some examples and Tom's link.
